public/app/percona/shared/core/reducers/dbaas/updateDBCluster/updateDBCluster.ts

```diff
@@ -0,0 +1,77 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+import { withAppEvents } from '../../../../../../features/alerting/unified/utils/redux';
+import { DBCluster } from '../../../../../dbaas/components/DBCluster/DBCluster.types';
+import { newDBClusterService } from '../../../../../dbaas/components/DBCluster/DBCluster.utils';
+import { DBClusterTopology } from '../../../../../dbaas/components/DBCluster/EditDBClusterPage/DBClusterAdvancedOptions/DBClusterAdvancedOptions.types';
+
+import { PerconaUpdateDBClusterState } from './updateDBCluster.types';
+
+export const initialUpdateDBClusterState: PerconaUpdateDBClusterState = {
+  result: undefined,
+  loading: undefined,
```

Comment:
Is there a difference when handling undefined and false for the loading prop?

Reply:
yes, undefined show that this state is default state before any loading, it can be useful, when you need to check some props or load something just for first time user see the page, but not for all next views, so if this false in store we know that we already tried to load something. In this case as I remember it's not necessary, but in my opinion it's a good when your default state is different from other negative values